app/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
 
     defaultConfig {
         applicationId "com.termux.api"
         minSdkVersion 24
-        targetSdkVersion 28
+        targetSdkVersion 29
```

Comment:
**Remove SDK 29 from there.**

Termux is not yet compatible with it and since uses a shared user id, that may lead to issues. https://github.com/termux/termux-packages/wiki/Termux-and-Android-10

Reply:
Welp, completely forgot about that.
F for android 10 termux package management BTW, hope everything turns out smooth for the users!
❤️